db/models/fund.py

```diff
@@ -8,13 +9,20 @@
 from sqlalchemy.orm import Mapped
 from sqlalchemy.orm import relationship
 from sqlalchemy.types import Boolean
+from sqlalchemy.types import Enum as SQLAEnum
 
 from db import db
 from db.models.round import Round
 
 BaseModel: DefaultMeta = db.Model
 
 
+class FundingType(Enum):
+    COMPETITIVE = "COMPETITIVE"
+    UNCOMPETED = "UNCOMPETED"
+    EOI = "EOI"
```

Comment:
is EOI a funding type like COMPETITIVE and UNCOMPETED ?

Reply:
Good question - I think one for the ticket!

Reply:
For these purposes, yes. Ideally I think we would remodel funds/rounds to account for EOI but we can't right now